examples/cloud-router-marketplace-susbscription/main.tf

```diff
@@ -0,0 +1,41 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+data "equinix_fabric_market_place_subscription" "test" {
```

Comment:
Let's use some different names than test in our examples. Try to be descriptive with customer facing changes.